src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -331,7 +331,15 @@ private function parseDefinition($id, $service, $file, array $defaults)
             }
         }
 
-        $tags = isset($service['tags']) ? $service['tags'] : array();
```

Comment:
I usually prefer `if`/`else` than the ternary operator but it's probably because I need glasses to read on a screen.
I'll change that.